### DISCUSSION TITLE
Seconds on taskbar clock?

### DISCUSSION THREAD

**Discussion by AaronKelley** [valinet/ExplorerPatcher#115] [Q&A] [answered]
This is amazing.  I hope that Microsoft takes note of everyone hating the Windows 11 taskbar and takes action to address it...

Anyway, I'm still on Windows 10 so I have a question.  With ExplorerPatcher in place, will the taskbar clock show **seconds** if [configured to do so](https://www.howtogeek.com/325096/how-to-make-windows-10s-taskbar-clock-display-seconds/)?  (The Windows 11 taskbar does not respect this registry setting.)  It might seem silly but this is one of the main things putting me off from Windows 11.

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
Yes seconds can be shown. Either using the Registry tweak or using 7+TT.

**Comment by valinet** [comment]
Also, I will add that as an option in the GUI, since I just tested it and it works only with that simple registry entry. Nice, did not even know such a thing was built-in, they never bothered to advertise it as far as I know. Thanks.
